src/rws_interface.cpp

```diff
@@ -374,6 +374,10 @@ bool RWSInterface::endSubscription()
   return rws_client_.endSubscription().success;
 }
 
+void RWSInterface::closeSubscription() {
```

Comment:
```suggestion
void RWSInterface::forceCloseSubscription()
{
```
Name suggestion as previous, and curly bracket on new line to be consistent with the rest of the code base.